pages/1.9/installing/ent/custom/cli/index.md

```diff
@@ -107,49 +107,51 @@ In this step, you create a YAML configuration file that is customized for your e
 
 1.  From your `genconf` directory, create a configuration file and save as `config.yaml`.
     
-    You can use this template to get started. This template specifies three masters, five [private](/1.9/overview/concepts/#private-agent-node) agents, one [public](/1.9/overview/concepts/#public-agent-node) agent, static master discovery list, an optional custom proxy, and SSH configuration specified. If your servers are installed with a domain name in your `/etc/resolv.conf`, you should add `dns_search` to your `config.yaml` file. For parameters, descriptions, and configuration examples, see the [documentation][3].
+    You can use this template to get started. This template specifies three masters, five [private](/1.9/overview/concepts/#private-agent-node) agents, one [public](/1.9/overview/concepts/#public-agent-node) agent, static master discovery list, and SSH configuration. You may use additional configuration parameters. For descriptions of all parameters and configuration examples, see the [documentation][6].
     
+    The CLI installer uses these default configuration values, which you may override in your configuration:
+
+    - `ssh_port`: `22`
+      - Only override this value if SSH on cluster nodes is available at a different port.
+    - `process_timeout`: `120`
+
     **Tips:** 
     
     - If Google DNS is not available in your country, you can replace the Google DNS servers `8.8.8.8` and `8.8.4.4` with your local DNS servers.
```

Comment:
@branden Line #120 should be removed as seen in /1.11/installing/ent/custom/cli/index.md